### DISCUSSION TITLE
SITE_OWNER question

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6509] [Firefly III] [answered]
What does setting SITE_OWNER actually do in the .env file?

**Comment by JC5** [comment, marked as answer]
It sets your email address in the From: field in messages that Firefly III can send and it's used in some error messages.

**Reply by Solverz-0** [reply]
Thank you, why would it be used in an error message? Out of interest?

**Reply by Solverz-0** [reply]
Also wouldn't the MAIL_FROM be the email that is used in the From: field?

**Reply by JC5** [reply]
Oh then I remembered it wrong. But it's used on the currency page for sure.

**Reply by Solverz-0** [reply]
Oh okay, so it should be like an admin sort of email?

**Reply by JC5** [reply]
Exactly.
